### PR TITLE
fix: sfselect fix v-model and stories

### LINKED ISSUE #1485: [BUG]: SfSelect v-model doesn't work
**Describe the bug**
You can not use v-model in SfSelect component. 

**How to reproduce**
1. Add v-model to SfSelect component
2. Try to change the option in select.
3. the error appears

**Expected behavior**
There should be working v-model. 

**Actual behavior**
Error appears after selecting an option.

**Code examples**
Add model object:
model: {
 props: ""
 event: ""
} 

**Screenshots**
https://files.slack.com/files-pri/T02FJEJ0A-F01CV18S4JH/image.png

**Devices (please complete the following information):**
 - Device: [e.g. iPhone6, Desktop]
 - Browser: [e.g chrome, safari][version]

**Additional context**
Anything you would like to add but was not sure which section it fits. For example: tech stack used (Nuxt, Vue, Gridsome, React, etc)

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes #1485

# Scope of work
Removed v-model on native select and added passing and receiving values from it. 
Storybook stories changed accordingly. 

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed
- [x] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/molecules/SfSelect/SfSelect.vue [hunk 1]
```diff
@@ -95,10 +102,10 @@ export default {
       default: "",
     },
   },
-  data() {
```
Comment (justyna-13) [inline packages/vue/src/components/molecules/SfSelect/SfSelect.vue:98]: There's an error in the console about mutating a prop directly so maybe data shouldn't be deleted, wdyt @AdamPawlinski ? :)

File: packages/vue/src/components/molecules/SfSelect/SfSelect.stories.js [hunk 2]
```diff
@@ -49,25 +50,39 @@ storiesOf("Molecules|Select", module)
...
+        ),
       },
     },
     data() {
       return {
-        selected: "",
         options: optionsList,
       };
     },
+    methods: {
```
Comment (justyna-13) [inline packages/vue/src/components/molecules/SfSelect/SfSelect.stories.js:70]: do we need this? :)

**Review by justyna-13 — COMMENTED** [review]
See my comments @AdamPawlinski  :)

File: packages/vue/src/components/molecules/SfSelect/SfSelect.vue (hunk 1 above)
Comment (AdamPawlinski) [reply packages/vue/src/components/molecules/SfSelect/SfSelect.vue:98]: It is connected with storybook, so it is on the dev side to use it correctly. Of course we should remove it finally, but now I consider it less important. I tried to add additional computed property for that, but it caused other errors, so I left it like this for now.

**Review by justyna-13 — APPROVED** [review]

**Review by Fifciu — APPROVED** [review]

**Pull request merged**